.circleci/config.yml

```diff
@@ -17,14 +17,10 @@ jobs:
       - run:
           name: Run tests
           command: yarn test
-  node-12:
+  node-14:
```

Comment:
A few things:
1. You'll need to update the circle config to use SF-CLI-BOT in order to do a successful release
2. You'll also need to update the job to publish this with the `core-v1` tag
3. You might also want to consider merging this into a `v4` branch instead of `main`. We don't have to do that but it would be easier if for any reason we need to release a change to `latest`

Reply:
@mdonnalley would you be against just manually publishing this for now instead of adding it to our automated CircleCI process?

I do plan on publishing it with a `4.0.0` version, `core-v1` tag instead of `latest` and I re-targeted a `v4` branch instead of `main`.

Reply:
@RodEsp I would prefer to use our release process if possible. Is there a particular reason you want to do it manually?

Reply:
@RodEsp Another thing to consider is that we've been putting all oclif/core related changes have been going into the `main` (as opposed to versioned branches like `v4`) and keeping `master` as the primary branch until oclif/core is GA, at which point we'd replace `master` with `main`

My preference would be to keep that pattern here

Reply:
> @RodEsp I would prefer to use our release process if possible. Is there a particular reason you want to do it manually?

@mdonnalley I guess I'm just fuzzy on where exactly `oclif` falls right now in terms of ownership and standardization. Maybe I missed it or wasn't here yet but was it decided `oclif` should have the same standards and ownership as the `salesforcecli` org?

> @RodEsp Another thing to consider is that we've been putting all oclif/core related changes into `main` (as opposed to versioned branches like `v4`) and keeping `master` as the primary branch until oclif/core is GA, at which point we'd replace `master` with `main`. You can see [oclif/plugin-not-found](https://github.com/oclif/plugin-not-found) for an example of this
> 
> My preference would be to keep that pattern here

Ah, got'cha. Sorry, I wasn't aware of that pattern. I'll fix that.

Reply:
Alrighty @mdonnalley, I updated the CircleCI `config.yml`, added the SF-CLI-BOT to the repo, as well as the corresponding env vars to the CircleCI build.